analysis/test_cases.go

```diff
@@ -4,25 +4,24 @@ package analysis
 // List of applications with expected analysis outputs.
 var Tier0TestCases = []TC{
 	TackleTestappPublicWithDeps,
-	TackleTestappPublicPackageFilter,
 	Tomcat,
-	CoolstoreWithDeps,
 	CoolstoreWithDepsQuarkus,
+	DaytraderWithDeps,
 	AcmeairWebappBinary,          // Binary upload
-	AdministracionEfectivoBinary, // Binary upload
 	TackleTestappPublicBinary,    // Binary fetched from tackle-testapp-public maven registry
 }
 
 // Tier 1 Analysis test cases - should work.
 // List of applications with expected analysis outputs.
 var Tier1TestCases = []TC{
-	// Setting empty until have working applications.
+	TackleTestappPublicPackageFilter,
+	CoolstoreWithDeps,
```

Comment:
CoolstoreWithDeps is important as we use it in Kai. Also, Package Filter as thats the only thing testing package filter functionality. I would  move CoolstoreWithDepsQuarkus out instead.